ansible/roles/cassandra-deploy-decrypt/tasks/main.yml

```diff
@@ -1,10 +1,10 @@
 - name: Copy the jar
   become: yes
   copy:
-    src: "{{cassandra_jar_path}}/user-declaration-util-1.0-SNAPSHOT-jar-with-dependencies.jar"
+    src: "{{cassandra_jar_path}}/{{cassandra_jar_name}}"
     dest: "{{cassandra_deploy_path}}"
   run_once: true
 
 - name: Run the application
-  shell: java -jar user-declaration-util-1.0-SNAPSHOT-jar-with-dependencies.jar
+  shell: java -jar {{cassandra_jar_name}}
```

Comment:
keep the entire value in double quotes @sknirmalkar89

Reply:
updated.